spec/suites/map/handler/Map.DragSpec.js

```diff
@@ -324,30 +338,34 @@ describe("Map.Drag", function () {
 		});
 
 		it.skipIfNotTouch("change the center of the map", function (done) {
-			map = new L.Map(container, {
+			map = new MyMap(container, {
 				dragging: true,
 				inertia: false
 			});
 			map.setView([0, 0], 1);
 
+			var start = L.point(200, 200);
+			var offset = L.point(256, 32);
+			var finish = start.add(offset);
+
 			var hand = new Hand({
 				timing: 'fastframe',
 				onStop: function () {
-					var center = map.getCenter();
-					var zoom = map.getZoom();
-					expect(center.lat).to.be.within(21.9430, 21.9431);
-					expect(center.lng).to.be(-180);
-					expect(zoom).to.be(1);
+					expect(map.getOffset()).to.eql(offset);
+
+					expect(map.getZoom()).to.be(1);
+					expect(map.getCenter().equals({ // small margin of error allowed
+						lat: 21.943045533, // 21.94304553343818
+						lng: -180
```

Comment:
Interesseting that it is still exact `-180`, in my tests the lng was not any more exact at -180, because the first move is not ignored anymore. But maybe I was wrong or you have changed something that I don't see.

Reply:
May be because of removing anim frame.